test/modern_bpf/test_suites/syscall_exit_suite/copy_file_range_x.cpp

```diff
@@ -0,0 +1,51 @@
+#include "../../event_class/event_class.h"
+
+#ifdef __NR_copy_file_range
+TEST(SyscallEnter, copy_file_rangeX)
```

Comment:
`TEST(SyscallEnter` <--- should be SyscallExit
(I wondered why the test did not show up :)

Reply:
yeah, the test is correct but in the wrong suite :man_facepalming: Great catch!!